.github/workflows/AutomateRelease.yml

```
@@ -0,0 +1,299 @@
+name: Automate release
+
+env:
+  PROJECT_KEY: "SONARPHP"
+  PROJECT_NAME: "SonarPHP"
+  LANGUAGE: "php"
+  USE_JIRA_SANDBOX: false
+  DRAFT_RELEASE: false
```

```suggestion
  IS_DRAFT_RELEASE: false
```
Preferences I guess, but that's how I would name it

---

Makes sense, I'll update it.